### ISSUE TITLE
article reference

### ISSUE THREAD

**Issue by jad-haddad** [haritha91/C3D-with-Batch-Normalization#1]
Hello,
I'm writing an article about emotion recognition in videos, and I based my network architecture on this. I searched your papers and didn't find one that talks about the current architecture, how can I reference your work in my paper?

**Comment by haritha91** [comment]
You can refer both the C3D paper and batchNorm paper in your article. It would be fine. My code is just a combination of those two.

**Comment by jad-haddad** [comment]
thank you for your answer :)
